solr/conf/schema.xml

```diff
@@ -428,6 +428,9 @@
     See http://wiki.apache.org/solr/SpatialSearch
    -->
     <fieldtype name="geohash" class="solr.GeoHashField"/>
+    
+    <fieldType name="_nest_path_" class="solr.NestPathField" />
```

Comment:
👍 I'm really excited to see this introduced.

Reply:
Me too!

![](https://media.giphy.com/media/Is1O1TWV0LEJi/giphy.gif)

I think there might be some exciting implications for GeoBlacklight too!